core/src/utils/test/ready.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { componentOnReady } from '../helpers';
+
+describe('componentOnReady()', () => {
+  it('should correctly call callback for a custom element', (done) => {
+    customElements.define('hello-world', class extends HTMLElement {
+      constructor() {
+        super();
+      }
+    });
+
+    const component = document.createElement('hello-world');
```

Comment:
nit: use "eager-component" / "lazy-component" instead of "hello-world" ?

Reply:
The lazy vs non lazy difference is already noted in the name of the test, so not sure this change would make much of a difference.